config/api.yml

```diff
@@ -2311,13 +2311,17 @@
         :identifier: physical_storage_new
       - :name: refresh
         :identifier: physical_storage_refresh
+      - :name: delete
+        :identifier: physical_storage_delete
```

Comment:
Believe you need to update the `:verbs` to include `delete` so `:verbs: *gpd`

Reply:
done